src/app/components/keyboard/keyboard.component.html

```diff
@@ -11,7 +11,7 @@
             </button>
         </div>
         <div class="row justify-content-between text-center">
-            <button class=" col-1 key" *ngFor="let key of thirdKeyBoardRow" (click)="pressKeyValue(key)" >
+            <button class="col-1" *ngFor="let key of thirdKeyBoardRow" (click)="pressKeyValue(key)" [disabled]="(key === '➜' && !disableKeys)" [ngClass]="(key==='➜' && !disableKeys)?'keyDisabled':'key'" >
```

Comment:
```suggestion
            <button class="col-1" *ngFor="let key of thirdKeyBoardRow" (click)="pressKeyValue(key)" [disabled]="disableKey(key)" [ngClass]="keyStyles(key)" >
```

Saca las comprobaciones al controlador

`disableKey(key: string): boolean { return key === '➜' && !this.disableKeys}`

`keyStyles(key: string): boolean { return (key==='➜' && !disableKeys)?'keyDisabled':'key' }`